doc/user_guide/Custom_Interactivity.rst

```diff
@@ -2,4 +2,5 @@ Custom Interactivity
 ____________________
 
 .. notebook:: holoviews ../../examples/user_guide/12-Custom_Interactivity.ipynb
+    :skip_execute: True
```

Comment:
I've added the print output to the notebook for now. Could you explain ``skip_output`` again? What specific line does it look for again?